src/Microsoft.ML.Transforms/TermLookupTransform.cs

```diff
@@ -22,7 +21,7 @@
 [assembly: LoadableClass(TermLookupTransform.Summary, typeof(TermLookupTransform), null, typeof(SignatureLoadDataTransform),
     "Term Lookup Transform", TermLookupTransform.LoaderSignature)]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.Text
```

Comment:
It should be in same namespace as Term transform.
Which is right now is Microsoft.ML.Transforms.Categorical.